docs/cicd/cicd-spans.md

```diff
@@ -0,0 +1,106 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Spans
+--->
+
+# Semantic conventions for CICD spans
+
+**Status**: [Development][DocumentStatus]
+
+<!-- toc -->
+
+- [CICD Spans](#cicd-spans)
+  - [Pipeline run](#pipeline-run)
+  - [Pipeline task run](#pipeline-task-run)
+
+<!-- tocstop -->
+
+## CICD Spans
+
+The conventions described in this section are specific to Continuous Integration / Continuous Deployment (CICD) systems.
+
+**Disclaimer:** These are initial CICD spans and attributes
+but more may be added in the future.
+
+Also consider setting other attributes of the [CICD and VCS resource conventions][cicdres].
```

Comment:
IMO If we think those are valuable on the spans here, we should add them here with proper requirement levels instead of being generic like this.

Reply:
Let's reword this to make the intent clearer.

We want to reference the relevant resource conventions to be used:
> Any resources of the [CICD and VCS resource conventions] that apply SHOULD be used.

Do we want to allow setting the attributes from the resources as span attributes as well (eg. if for technical reasons they cannot be set as resources)?
We could add the following:
> If resources cannot be used for technical reasons, the resource attributes MAY be set as span attributes.